### PR TITLE
Acceptance test Wiki page update.

### PR DESCRIPTION
**Pull request by Akhilesh-max**
Directory structure in the wiki page has been updated, and additional tips and notes have been included.

### CODE REVIEW

**Comment by Akhilesh-max** [comment]
@rahat2134 @jnvtnguyen PTAL.

**Comment by Akhilesh-max** [comment]
@rahat @jnvtnguyen Please take a look.

Thanks...

**Comment by jnvtnguyen** [comment]
Sorry will take a look by EOD, Thanks.

**Review by jnvtnguyen — APPROVED** [review]
LGTM, Thanks.

**Comment by Akhilesh-max** [comment]
@seanlip  PTAL, 
It looks like, your review is needed herein. 

Thanks.

File: Acceptance-Tests.md [hunk 1]
```diff
@@ -114,20 +151,24 @@ make run_tests.acceptance suite="blog-editor-tests/check-blog-editor-unable-to-p
...
+9) The `utilities` directory contains all the utility files and helper functions, which you would require to write new acceptance tests. This directory can also be used to append more utility functions as and when required or needed by the user.
 
 10) The test must be thoroughly tested before submitting a PR. The test can be run locally by running the following command as mentioned above or you can run the test on the CI server by pushing your code to the remote branch in your fork. The CI server will run the test and will show the result.
 
+11) After writing the test, do not forget to add it in our configuration file `common.py` and in `acceptance.json` file so that it is included in the workflow.
+
+> Note: Sometimes tests may pass locally but fail on the CI environment due to differences between the local and CI environments. In such cases, debugging and fixing should be done on the CI environment, as that is where the tests are intended to run. However, We are transitioning to using Docker for both local and CI setups, which should help mitigate these issues.**
```
Comment (seanlip) [inline Acceptance-Tests.md]: However, We --> However, we

both local and CI setups --> both the local and CI setups

File: Acceptance-Tests.md [hunk 2]
```diff
@@ -114,20 +151,24 @@ make run_tests.acceptance suite="blog-editor-tests/check-blog-editor-unable-to-p
       } as const;
     ```
 
-    For roles that don't require super admin privileges, such as `LoggedInUser`, add the factory to the array inside `createNewUser` under `composeUserWithRoles(BaseUserFactory(), [...])`. This ensures that the new user role is included when creating a new user instance.
-
-    Please ensure to follow the appropriate conventions and guidelines while adding new user-utilities files to the user-factory to maintain consistency and clarity in the testing process.
+For roles that don't require super admin privileges, such as `LoggedInUser`, add the factory to the array inside `createNewUser` under `composeUserWithRoles(BaseUserFactory(), [...])`. This ensures that the new user role is included when creating a new user instance.
+
```
Comment (seanlip) [inline Acceptance-Tests.md]: Drop the spaces here

**Review by seanlip — APPROVED** [review]
Thanks @Akhilesh-max! I noticed a few small things but feel free to merge after fixing them. Thanks very much for updating this page!

File: Acceptance-Tests.md (hunk 1 above)
Comment (Akhilesh-max) [reply Acceptance-Tests.md]: Done.

File: Acceptance-Tests.md (hunk 2 above)
Comment (Akhilesh-max) [reply Acceptance-Tests.md]: Done.

**Comment by Akhilesh-max** [comment]
@seanlip PTAL. 

Apparently, I can't merge this myself. 

Screenshot 2024-08-19 at 8 43 04 PM

**Comment by seanlip** [comment]
Thanks @Akhilesh-max! Merging, but I think the permissions issue is also fixed now (I just updated one of the settings).

**Pull request merged**